dashboard/test/ui/browsers.json

```diff
@@ -21,7 +21,7 @@
     "name": "Safari",
     "browserName": "Safari",
     "browserVersion": "16",
-    "platformName": "macOS 11.00"
+    "platformName": "macOS 12.00"
```

Comment:
```suggestion
    "platformName": "macOS 12"
```

Reply:
Per https://saucelabs.com/products/platform-configurator#/ the value is not 12.00

Reply:
yep that seems right